nengo_gui/namefinder.py

```diff
@@ -22,16 +22,17 @@ def find_names(self, net):
         classes = (nengo.Node, nengo.Ensemble, nengo.Network,
                    nengo.Connection)
 
-        for k in dir(net):
-            if not k.startswith('_') and k not in base_lists + all_lists:
-                v = getattr(net, k)
-                if isinstance(v, list):
-                    for i, obj in enumerate(v):
+        for class_attr in dir(net):
```

Comment:
Hmm, I'm not sure about ```class_attr``` as a variable name.  To me, that seems to indicate that these are class attributes, but they aren't; they're instance attributes.